index.html

```diff
@@ -59,6 +59,170 @@ <h2 class="primary-header green">
+          </div>          
+        </article>
+  
+        <article class="card">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
+          </div>          
+        </article>
+  
+        <article class="card">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
+          </div>          
+        </article>
+  
+        <article class="card">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
+          </div>          
+        </article>
+      </div>
+      
+    </section>
+
+    <section class="headline" id="about-me">
+      <div class="about-me-intro">
+        <h1 class="primary-header">About me</h1>
+        <p id="about-me-paragraph">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded,
+          don’t hestiate to contact me.
+        </p>
+        <button class="green-button" id="resume-button">Get My Resume</button>
+      </div>
+      <article class="about-me-item">
+        <header><img class="about-me-items-image" src="assets/images/icon-languages.svg" alt="Language"></header>
```

Comment:
- [ ] I don't see any use in wrapping your image (logo) inside a `<header>` tag, to make your code cleaner and more professional I would suggest removing the header tag from all the logos of the technologies